src/Approximations/overapproximate.jl

```diff
@@ -847,7 +872,7 @@ julia> center(Z)
 3-element Array{Float64,1}:
   1.0
  -2.1
-  0.8999999999999999
+  2.4
```

Comment:
So the previous example was broken. Do you know whether the new version is correct?

Reply:
i don't know when did the result change.